StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodMessagingElement/PMME+Internal.swift

```diff
@@ -79,27 +91,29 @@ extension PaymentMethodMessagingElement {
             ).first else {
                 // This should never happen, but if it does we log an error and attempt to fall back to a multi-partner style
                 //      (so that we can use the promotion text, which doesn't require a logo, instead of inline) without logos
-                stpAssertionFailure("No images returned by API")
-                // TODO(gbirch) add error analytics
+                _ = Self.assertAndLogMissingField("logo", apiClient: configuration.apiClient)
```

Comment:
image could fail to load for expected reasons too. Not sure what to do, maybe we should only consider 'request completed w/ non-200 http status code' as unexpected or something.